gateway/src/apicast/backend/cache_handler.lua

```diff
@@ -34,28 +34,29 @@ local function fetch_cached_key()
   return ok and stored
 end
 
-function _M.handlers.strict(cache, cached_key, response, ttl)
-  if response.status == 200 then
+function _M.handlers.strict(cache, cached_key, status_code, ttl)
```

Comment:
This is breaking change for someone who written a custom handler, right?

Reply:
Yes. Didn't think about users overridding this method.